src/Features/CustomerEffortScoreTracks.php

```diff
@@ -114,10 +133,27 @@ public function run_on_transition_post_status(
 		$old_status,
 		$post
 	) {
-		if ( 'product' !== $post->post_type ) {
+		if ( ! in_array( $post->post_type, self::TRACKED_POST_TYPES, true ) ) {
```

Comment:
This `if` clause seems a little redundant with the below conditional.

Reply:
Agreed! I will remove this check before I merge.